File: Java_files/calculator.java
import java.util.Scanner;

public class SimpleAdditionCalculator {
    public static void main(String[] args) {
        // Create a scanner object to take input from the user
        Scanner scanner = new Scanner(System.in);

        // Take the first number as input
        System.out.print("Enter the first number: ");
        double num1 = scanner.nextDouble();

        // Take the second number as input
        System.out.print("Enter the second number: ");
        double num2 = scanner.nextDouble();

        // Perform addition
        double result = num1 + num2;

        // Display the result
        System.out.println("The result of addition is: " + result);
        
        // Close the scanner
        scanner.close();
    }
}

File: Python_files/calculator.py
# Function to add two numbers
def add_numbers(num1, num2):
    return num1 + num2

# Main program
def main():
    print("Welcome to the Simple Calculator!")

    # Get user input
    try:
        number1 = float(input("Enter the first number: "))
        number2 = float(input("Enter the second number: "))
        
        # Calculate the sum
        result = add_numbers(number1, number2)
        
        # Display the result
        print(f"The sum of {number1} and {number2} is: {result}")
    except ValueError:
        print("Please enter valid numbers.")

# Run the calculator
if __name__ == "__main__":
    main()
